verify.ts

```diff
@@ -0,0 +1,265 @@
+    }
+  }
+}
+
+const processContracts = async (): Promise<any> => {
+  let exitCode = 1;
+  let anySucceeded = false;
+  let spinner: any;
+
+  try {
+
+    // tslint:disable-next-line: max-line-length
+    const etherscanUrl = `https://api${options.network === 'main' ? '' : ('-' + options.network)}.etherscan.io/api`;
+
+    // const flattenedFake = fs.readFileSync('./flattened.sol', 'UTF-8');
+
+    if (options.check) {
+      // check is the GUID
+      const result = await validateResult(options.check, etherscanUrl);
+      console.log(`Verification status: ${result.result}`);
+      exitCode = 0;
+    } else {
+
+      console.log(`Verifying contract(s) on ${options.network}...`);
+      // tslint:disable-next-line: forin
+      for (const contractName in addresses) {
+
+        let constructorArguments: Buffer;
+        switch (contractName) {
+          case 'GEN':
+          case 'ControllerCreator':
+            continue;
+
+          case 'GenesisProtocol':
+            constructorArguments = abi.rawEncode(['address'], [allAddresses.GEN]);
+            break;
+          case 'Redeemer':
+            constructorArguments =
+              abi.rawEncode(['address', 'address'], [allAddresses.ContributionReward, allAddresses.GenesisProtocol]);
+            break;
+          case 'DaoCreator':
+            constructorArguments =
+              abi.rawEncode(['address'], [allAddresses.ControllerCreator]);
+            break;
+          case 'DAORegistry':
+            constructorArguments =
+              abi.rawEncode(['address'],
+                [options.network === 'main' ? '0xd3BA32dd207Db75f535001FAC749c925423D8A6f' :  // DAOstack multisig :
+                  '0x73Db6408abbea97C5DB8A2234C4027C315094936']);
+            break;
+        }
+
+        spinner = ora();
+        spinner.start(`${contractName} at ${addresses[contractName]}`);
+
+        let foundIn = './node_modules/@daostack/arc';
+        process.chdir(path.join(__dirname, foundIn));
+
+        let solFilePath = glob.sync(`./contracts/**/${contractName}.sol`);
+        if (!solFilePath.length) {
+          foundIn = './node_modules/@daostack/arc/node_modules/@daostack/infra';
+          process.chdir(path.join(__dirname, foundIn));
+          solFilePath = glob.sync(`./contracts/**/${contractName}.sol`);
+        }
+
+        if (!solFilePath.length) {
+          foundIn = './node_modules/@daostack/arc-hive';
+          process.chdir(path.join(__dirname, foundIn));
+          solFilePath = glob.sync(`./contracts/**/${contractName}.sol`);
+        }
+
+        if (!solFilePath.length) {
+          spinner.fail(`contract ${contractName}.sol not found`);
+          continue;
+        }
+
+        const flattened = await truffleFlattener([solFilePath[0]]);
+
+        if (options.outputFlattened) {
+          fs.writeFileSync(path.join(options.outputFlattened, `flattened.${contractName}.sol`), flattened);
+        }
+
+        // tslint:disable-next-line: max-line-length
```

Comment:
why to disable that here?

Reply:
i removed it and split the following line.